app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js

```diff
@@ -9,20 +9,23 @@ describe('associated-services order-item contextCreator', () => {
     const orderId = 'order-id';
     const mockAssociatedServicePrices = { prices: [{ priceId: 29 }, { priceId: 30 }] };
     const somefakeUrl = 'https://some.url.co.uk/order-id';
+    const odsCode = '03F';
     it.each`
     senderUrl                               |  expectedUrl                           | associatedServicePrices
     ${`${somefakeUrl}/associated-service`}  | ${`${somefakeUrl}/associated-service`} | ${''}   
     ${`${somefakeUrl}/price`}               | ${`${somefakeUrl}/price`}              | ${''}  
     ${`${somefakeUrl}/associated-services`} | ${`${somefakeUrl}/associated-services`}| ${''}   
-    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}   
+    ${`${somefakeUrl}/neworderitem`}        | ${`${baseUrl}/organisation/${odsCode}/order/${orderId}/associated-services/select/associated-service/price`} |${mockAssociatedServicePrices}   
```

Comment:
Can we fix the alignment of the vertical bars in the table?

Reply:
done